src/index.js

```diff
@@ -0,0 +1,44 @@
+import './style.scss';
+import init from './modules/domElements';
+import createTodoInstance from './modules/createTodo';
+import {
+  validateForm,
+  validateProject,
+  initProject,
+  refreshProjects,
+} from './modules/helpers';
+import displayTodos, { projectEvent } from './modules/displayTodos';
+import createProject from './modules/createProject';
+
+initProject();
+init();
+projectEvent();
+
+const todoForm = document.querySelector('#todo-form');
+const submit = document.querySelector('#create-todo');
+const createProjectBtn = document.querySelector('#create-project');
+const projectForm = document.querySelector('#project-form');
+
+submit.addEventListener('click', e => {
+  e.preventDefault();
+  const { projects } = todoForm.elements;
+  const groupTitle = document.getElementById('group-title');
+  if (validateForm(todoForm)) {
+    createTodoInstance(todoForm);
+    groupTitle.textContent = String(projects.value) === 'allTodos' ? 'Home' : projects.value;
+    displayTodos(String(projects.value));
+    todoForm.reset();
+  }
+  return null;
+});
+
+createProjectBtn.addEventListener('click', e => {
+  e.preventDefault();
+  if (validateProject(projectForm)) {
+    createProject(projectForm);
+    refreshProjects();
+    projectEvent();
+    projectForm.reset();
+  }
+  return null;
```

Comment:
Awesome validation features 👍 

I typed in 4 letters project name, and the button seems not working...

**It would be nice if you could have an error pop up message that will tell the user character too short.**